database/db.py

```diff
@@ -51,8 +51,8 @@ def dbFetch(self, statement):
             cursor = mydb.cursor()
             cursor.execute(statement)
             result = cursor.fetchall()
-        except Error as e:
-            logger.error(error)
+        except OSError as e:
```

Comment:
How come this is switched to oserror?

Reply:
I think Error is actually wrong... not sure how that got in there. The other ones use mysql.connector.Error.... maybe thats the one we want? It's been a while since I wrote those methods and I cant remember what exactly mysql will throw.